### DISCUSSION TITLE
官方文档错误

### DISCUSSION THREAD

**Discussion by mixpplus** [AlistGo/alist#2038] [General]
在 `安装-->手动安装` 中的**守护进程**章节中官网书写错误，希望进行一下改正
![image](https://user-images.githubusercontent.com/47939006/196033679-d91a1cce-efee-4d45-9a1e-97d9942003ea.png)

**Comment by anwen-anyi** [comment]
QAQ V2的吧这是？ V2的不需要server就能启动吖~
